modules/rosa-sharing-vpc-hosted-zones.adoc

```diff
@@ -16,6 +16,8 @@ image::372_OpenShift_on_AWS_persona_worflows_0923_3.png[]
 
 .Procedure
 
+. In the link:https://us-east-1.console.aws.amazon.com/ram/[Resource Access Manager of the AWS console], create a resource share that shares the previously created public and private subnets with the *Cluster Creator's* AWS account ID.
```

Comment:
I don't know if we can provide a region-agnostic link to this, but it would be good if we could - it seems to connect to your local region if you trim out the `us-east-1.` from the beginning, so it may be worth specifying that you need to change to the region you want to edit?